apps/storybook/src/Selection.stories.tsx

```diff
@@ -24,6 +27,13 @@ function vectorToStr(vec: Vector2) {
   return `(${format('.2f')(vec.x)}, ${format('.2f')(vec.y)})`;
 }
 
+const X = range(-5, 5, 0.01);
+const Y = X.map((x) => Math.exp((-x * x) / 0.2));
+
+function GaussianCurve() {
+  return <DataCurve abscissas={X} ordinates={Y} color="blue" />;
+}
```

Comment:
I made a new story to demonstrate the new component. I found it more user-friendly to have something to zoom at, hence this small component that displays a gaussian curve.